### PR TITLE
add Emscripten demo

### PR DESCRIPTION
**Pull request by SSE4**
related:
https://github.com/conan-io/docs/pull/1249
https://github.com/conan-io/conan/pull/4346

### CODE REVIEW

**Comment by danimtb** [comment]
Please include this example in the CI scripts

**Comment by SSE4** [comment]
@danimtb okay

**Comment by SSE4** [comment]
```
ERROR: Invalid setting 'wasm' is not a valid 'settings.arch' value.
Possible values are ['armv5el', 'armv5hf', 'armv6', 'armv7', 'armv7hf', 'armv7k', 'armv7s', 'armv8', 'armv8.3', 'armv8_32', 'avr', 'mips', 'mips64', 'ppc32', 'ppc64', 'ppc64le', 's390', 's390x', 'sparc', 'sparcv9', 'x86', 'x86_64']
Read "http://docs.conan.io/en/latest/faq/troubleshooting.html#error-invalid-setting"
```
should I patch settings.yml during the build? or just wait for the release?

**Comment by danimtb** [comment]
We will wait for the release then. I didn't notice the new settings.

Please add some documentation to a readme so people can understand the example and link it to the main readme page.

**Comment by SSE4** [comment]
added description to the readme

File: features/emscripten/build.cmd [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@echo off
```
Comment (danimtb) [inline features/emscripten/build.cmd]: The build script needs to be `build.bat`

Comment (SSE4) [reply features/emscripten/build.cmd]: renamed

File: features/emscripten/build.bat [hunk 2]
```diff
@@ -0,0 +1,5 @@
+@echo off
+conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
```
Comment (danimtb) [inline features/emscripten/build.bat]: Bincrafters remote is no longer needed, as `emsdk_installer` is already included in Conan Center
```suggestion
```

File: features/emscripten/build.sh [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -ex
+
+conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
```
Comment (danimtb) [inline features/emscripten/build.sh]: Same here
```suggestion
```

**Pull request merged**